packages/fxa-settings/src/components/Modal/index.tsx

```diff
@@ -78,11 +82,14 @@ export const Modal = ({
 
                 {onConfirm && (
                   <button
-                    className="cta-primary transition-standard flex-1"
+                    className={classNames(
+                      'cta-primary transition-standard flex-1',
+                      confirmBtnClassName
```

Comment:
I just noticed this uses `confirmBtnClassName` _and_ `cta-primary`, which would create competing styles. Can `confirmBtnClassName` just default to `cta-primary`?

Reply:
I like that much better :)